src/api/rest/server/server.go

```diff
@@ -128,8 +128,10 @@ func ApiServer() {
 	e.GET("/tumblebug/config", rest_common.RestGetAllConfig)
 	e.DELETE("/tumblebug/config", rest_common.RestDelAllConfig)
 
-	e.GET("/tumblebug/objectList", rest_common.RestGetObjectList)
-	e.GET("/tumblebug/objectValue", rest_common.RestGetObjectValue)
+	e.GET("/tumblebug/objects", rest_common.RestGetObjectList)
+	e.GET("/tumblebug/object", rest_common.RestGetObjectValue)
+	e.DELETE("/tumblebug/object", rest_common.RestDeleteObject)
+	e.DELETE("/tumblebug/objects", rest_common.RestDeleteObjects)
```

Comment:
```suggestion
	e.GET("/tumblebug/objects", rest_common.RestGetObjects)
	e.GET("/tumblebug/object", rest_common.RestGetObject)
	e.DELETE("/tumblebug/object", rest_common.RestDeleteObject)
	e.DELETE("/tumblebug/objects", rest_common.RestDeleteObjects)
```

일관성 차원에서.. 이렇게 하는 것은 어떨까 합니다.. ^^

---

[함께 바꿔야 하는 부분]
- `src/api/rest/server/common/utility.go`
  - `func RestGetObjectList(c echo.Context) error {`
  - `func RestGetObjectValue(c echo.Context) error {`

Reply:
@jihoon-seo 감사합니다. 리뷰 반영하였습니다~!